build/packaging/suseconnect-ng.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Apr 15 12:37:29 UTC 2024 - Guang Yee <gyee@suse.com>
+
+- Update to version 1.9.0
+  * Add the ability to upload the system uptime logs, produced by the suse-uptime-tracker daemon, to SCC/RMT as part of daily heartbeat report. 
```

Comment:
Please add tracking cards here, either from bugzilla (bsc) or jira (ped). Look at past entries for suggestions

Reply:
We have an epic on metering at PED-7982 , and the ECO at PED-8018... still working on the 3rd and final approval for the ECO if that matters.

Reply:
Thanks. I've added the two PEDs.